include/ffnvcodec/dynlink_loader.h

```diff
@@ -304,6 +337,19 @@ static inline int cuda_load_functions(CudaFunctions **functions, void *logctx)
     LOAD_SYMBOL(cuTexRefSetArray, tcuTexRefSetArray, "cuTexRefSetArray");
     LOAD_SYMBOL(cuTexRefSetFilterMode, tcuTexRefSetFilterMode, "cuTexRefSetFilterMode");
 
+    //more setting fore texref
+    GET_PROC(cuTexRefSetAddressMode);
```

Comment:
before we push this upstream we should probably apply these everywhere, otherwise its kinda messy

Reply:
ah, you mean the get_proc

Reply:
I think it is not possible to switch all of it to GET_PROC, since some of the symbol does not meet this pattern. For example: `cuvidMapVideoFrame64`